app/services/hyrax/characterizer.rb

```diff
@@ -0,0 +1,19 @@
+module Hyrax
+  ##
+  # @abstract Determins which characterizer to run based on the file_set type
```

Comment:
I'm not sure I'd consider this class to be an abstract class, so I'd be inclined to drop the `@abstract` tag.